crates/rolldown_testing/_config.schema.json

```diff
@@ -462,6 +472,23 @@
         }
       }
     },
+    "ESTarget": {
+      "type": "string",
+      "enum": [
+        "eS5",
```

Comment:
LoL. Looks like that's one of the reason for https://github.com/rolldown/rolldown/pull/2672/files#r1835472534